postgres_metrics/views.py

```diff
@@ -7,20 +7,18 @@
 
 
 def metrics_view(request, name):
-    if not request.user or not request.user.is_superuser:
+    if not request.user.is_superuser:
```

Comment:
What happens if `request.user` is undefined? Or can't this happen?

Reply:
It won't happen in a normal Django setup with auth enabled, as when there isn't an authenticated user, it will be an AnonymousUser.